packages/styling/src/styles/DefaultPalette.ts

```diff
@@ -10,6 +10,7 @@ export const DefaultPalette: IPalette = {
   themeLight: '#c7e0f4',
   themeLighter: '#deecf9',
   themeLighterAlt: '#eff6fc',
+  themePrimaryTransparent: 'rgba(0, 120, 215, 0.1)',
```

Comment:
don't add this to defaults (it shouldn't be needed since we'll always populate a default in theme.ts)